### PR TITLE
Feature/javascript popup

### PR DESCRIPTION
**Pull request by MahdiAghaali**
In this PR I did:

- Remove static project elements and add them dynamically using javascript
- Create a popup window and add functionality to it using javascript

- Fix font issues

### CODE REVIEW

File: Contents/Popup.css [hunk 1]
```diff
@@ -0,0 +1,98 @@
+
+#popup {
```
Comment (MahmoudBakr23) [inline Contents/Popup.css:2]: - [ ] Great work for designing the modal so far. However, you still have a few more adjustments to go before rocketing it 😉 
1. Please make sure to align the used techs sections beneath the project's name correctly
2. Kindly adjust the width and height of the project's image
3. Nice styling of the paragraph, but I'm afraid you will have to stick to the given design
_Take another look at [Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=23%3A9) design, and the picture down below of your project and make the changes done. 🙏 
Screenshot (1)

**Review by MahmoudBakr23 — CHANGES_REQUESTED** [review]
Hello @MahdiAghaali 😃, 

Great work and effort you put into this project so far.

## Highlights 🚀 

There are some excellent points that you have made perfectly. Keep up! 💪
- Well done with following the Figma design
- Great work with javascript
- Descriptive PR and Readme file
- Correct Linters

However, there is always room for improvement.

> Keep in mind that your long hours of work are appreciated. However, the code review process improves the code you wrote, not criticizes it. 😄

 ## Required Changes ♻️
_Please check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code._

N/A

Cheers and Happy coding!👏👨‍💻💻

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by sja-thedude — CHANGES_REQUESTED** [review]
## Required Changes ♻️

Hi @MahdiAghaali, This is @sja-thedude your code reviewer.

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

200w

 ## Highlights
✔️ Good job so far

 ## Required Changes ♻️

- [ ] Kindly make sure to implement all the changes asked by the previous reviewer. For this watch [this](https://www.loom.com/share/b4f6046705bf4e39a992d4658bc8a270) loom video.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me [@sja-thedude] in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by TedLivist — APPROVED** [review]
## STATUS: APPROVED ✔️ ✔️

Hi @MahdiAghaali,

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. And please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**